packages/expo-file-system/ios/FileSystemHelpers.swift

```diff
@@ -1,6 +1,10 @@
 // Copyright 2023-present 650 Industries. All rights reserved.
 
 import ExpoModulesCore
+import Photos
+
+private let assetIdentifier = "ph://"
+private let resourceManager = PHAssetResourceManager()
```

Comment:
nit: could we lazy initiate ths instance or even just using local variable in the usage?
```swift
PHAssetResourceManager().writeData(...)
```

Reply:
Good catch, we should make it lazy. We still need to keep it outside of the function because it has to live long enough for the callback to get executed.